first class/first.js

```diff
@@ -0,0 +1,44 @@
+console.log("connected with external js file");
+
+var a=1;
+var b=2;
+if(a+b>4)
```

Comment:
It will be better if we write these conditional parts as following the pattern below:

```javascript
if (condition1) {
  // inner expressions
} else if (condition2) {
  // inner expressions
} else {
  // inner expressions
}